typescript/keys.d.ts

```diff
@@ -22,14 +22,16 @@ export interface NewDeveloperKey extends BaseKey {
   }
 }
 
-export type NewKey = {
-  secret: string
-} & (NewLegacyKey | NewDeveloperKey)
+export interface NewPowerAutomateKey extends BaseKey {
+  type: 'POWER_AUTOMATE'
```

Comment:
is it worth calling this a generic webhook subscription type?

I can see the argument for different subscriptions needing different perms

Reply:
Zac and I (and I think Josh) agreed being more specific was better as it allows us to only show this type of key on the integrations page without any magic knowledge.